tests/test_processing_common.py

```diff
@@ -88,6 +91,10 @@ def prepare_image_inputs(self):
         image_inputs = [Image.fromarray(np.moveaxis(x, 0, -1)) for x in image_inputs]
         return image_inputs
 
+    @require_vision
+    def prepare_video_inputs(self):
+        return [np.random.randint(255, size=(4, 3, 30, 400), dtype=np.uint8)]
+
```

Comment:
not sure I understand this. In overriden tests we had a list of images (aka frames) and here we have a list of videos? Does that mean overriden tests can't process batched videos?

Reply:
I fixed the `make_batch` funcs in the image processor files instead. So, this aren't needed anymore.

Thanks for raising it!